File: strings/prob771.py
class Solution(object):
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        d = {j:0 for j in J}
        for s in S:
            if s in d:
                d[s] += 1
        return sum(d.values())
        

File: strings/prob771.kt
class Solution {
    fun numJewelsInStones(jewels: String, stones: String): Int {
        
        val js = jewels.toSet()
        var ans = 0
        for (c in stones) {
            if (c in js) {
                ans++
            }
        }
        
        return ans
    }
}
